generator/generator.py

```diff
@@ -0,0 +1,102 @@
+"""This modules generates and saves data in csv/excel file"""
+import csv
+import random
+import sys
+import xlsxwriter
+from recruitment_data import RecruitmentData
+from candidate_data import CandidateData
+
+
+def generate_data(recruitments_number):
+    """
+    Generates recruitment and candidates data
+    :param recruitments_number: Number of recruitments you want to generate
+    :return: List of generated RecruitmentData and CandidateData
+    """
+    recruitments = [RecruitmentData() for _ in range(recruitments_number)]
+    for recruitment in list(recruitments):
+        recruitments.append(RecruitmentData.from_previous_cycle(recruitment))
+    candidates = [CandidateData(data) for data in recruitments]
+    for recruitment in recruitments:
+        number_of_candidates = random.randrange(200)
+        for _ in range(number_of_candidates):
+            candidate = random.choice(candidates)
+            if candidate.can_be_next_recruitment(recruitment):
+                candidates.append(CandidateData.from_previous_recruitment(candidate, recruitment))
+            else:
+                candidates.append(CandidateData(recruitment))
+    return recruitments, candidates
+
+
+def generate_csv(recruitments, candidates):
+    """
+    Generates csv file and saves data in it
+    :param recruitments: List of RecruitmentData
+    :param candidates: List of CandidateData
+    """
+    with open("recruitments.csv", 'w') as csv_file:
+        writer = csv.writer(csv_file)
+        writer.writerow(["faculty", "degree", "major_name", "mode",
+                        "end_date", "cycle_number", "slot_limit"])
+        for recruitment in recruitments:
+            writer.writerow(list(recruitment))
+    with open("candidates.csv", 'w') as csv_file:
+        writer = csv.writer(csv_file)
+        writer.writerow(["pesel", "name", "city", "region", "country", "highschool",
+                         "highschool_city", "matura_date", "graduation_date", "matura_result",
+                         "test_result", "finished_college_name", "finished_faculty",
+                         "finished_field_of_study", "finished_mode", "average", "faculty",
+                         "degree", "major_name", "mode", "end_date", "cycle_number", "slot_limit"])
```

Comment:
@jmysliv maybe we should add this to **settings.py** or sth like that bcs we are going to use it somewere for sure. It is easy to make mistake here

Reply:
@twrdyyy I created get_dict method for RecruitmentData and CandidateData which solves that problem.